documentation/synthetic_tokens/tutorials/creating_from_sponsor_cli.md

```diff
@@ -0,0 +1,207 @@
+# UMA Sponsor CLI Tool
+
+The Sponsor CLI tool lets you interact with a synthetic token contract.
+In this tutorial, you’ll be deploying a synthetic token contract to a local testnet and interacting with it via the Sponsor CLI tool.
+
+It will let you be the “Token Sponsor,” which lets you:
+
+- Deposit collateral (ETH) and borrow synthetic tokens
+- Manage your position’s collateral (deposit/withdraw)
+- Deposit additional collateral
+- Withdraw excess collateral
+- Redeem the borrowed synthetic tokens and get collateral back
+- Transfer your position to another Ethereum address
+
+This [video](https://www.crowdcast.io/e/defi-discussions/18) will walk through the steps of this tutorial, starting around minute 11:00.
+
+## Prerequisites
+
+1. Install homebrew via website `brew.sh`
+2. Install [Node v12](https://nodejs.org/en/download/) and make sure that npm is v6 or above. You can verify by running in the terminal:
+
+```bash
+node -v
+npm -v
+```
+
+3. If on local testnet:
+
+- Install [Ganache](https://www.trufflesuite.com/ganache)
+- Run ganache
+- Click “new workspace”
+- In the server tab, change “Port Number” to `9545`
+- In the “Accounts & Keys” tab, change “Account Default Balance” to `100000000`
+- (Optional) In the Server tab, deselect "Automine". Turning off "Automine" provides a more “authentic” blockchain experience but can be very frustrating.
+
+Once you have set up your environment, clone the repo and install the CLI tool.
+
+4. Download the code by typing this into terminal:
+
+```bash
+git clone https://github.com/UMAprotocol/protocol.git
+```
+
+5. Run `npm install` in `protocol/`.
+
+## Launching the CLI tool
+
+1. Navigate to the `/core` folder: `cd core`
+2. If on Kovan testnet, apply network addresses:
+
+```bash
+$(npm bin)/apply-registry
+```
+
+3. Compile and migrate the contracts:
+
+```bash
+$(npm bin)/truffle compile
+$(npm bin)/truffle migrate --reset --network=test
+```
+
+4. Deploy a contract to create priceless synthetic tokens named “BTCUSD”.
+   Each synthetic token is an ERC-20 token that represents a synthetic bitcoin, collateralized by ETH.
+
+```bash
+$(npm bin)/truffle exec scripts/local/DeployEMP.js --network=test
+```
+
+This is the output you should see:
+
+![](deployEMP_output.png)
+
+5. Run the CLI.
+
+```bash
+sh scripts/cli/cli_entry.sh --network=test
```

Comment:
I'm pretty sure you can simply run `uma --network=test`. The package.json should automatically install `uma` as a command line tool you can use that does the same thing as `... cli_entry.sh`

Reply:
Look here for reference. Can you try this and see if you can get `uma` CLI installed?

https://github.com/UMAprotocol/protocol/blob/master/documentation/oracle/tutorials/voting_with_uma_tokens.md

Reply:
Great - this is addressed in this commit: https://github.com/UMAprotocol/protocol/pull/1365/commits/5b2825cd040118acd76ef055ad6a26578e014833